dev/test-studio/plugins/locale-no-nb/bundles/studio.ts

```diff
@@ -101,6 +101,108 @@ const studioResources: Record<StudioLocaleResourceKeys, string> = {
 
   /** Label for selecting a hour preset. Receives a `time` param as a string on hh:mm format and a `date` param as a Date instance denoting the preset date */
   'inputs.datetime.calendar.action.set-to-time-preset': '{{time}} on {{date, datetime}}',
+
+  /** --- File (Image, File and ImageTool) Inputs --- */
+
+  /** Open image edit dialog */
+  'inputs.files.image.actions-menu.edit-details.label': 'Åpne bilde redigeringsdialog',
```

Comment:
Særskrivingsfeil, tror jeg?

```suggestion
  'inputs.files.image.actions-menu.edit-details.label': 'Åpne bilderedigeringsdialog',
```

Reply:
Særskrivingsfeil, helt klart.